frontend/app/routes/protected/person-case/request-details.tsx

```diff
@@ -0,0 +1,157 @@
+      type: context.session.inPersonSINCase?.requestDetails?.type,
+      scenario: context.session.inPersonSINCase?.requestDetails?.scenario,
+    },
+  };
+}
+
+export function meta({ data }: Route.MetaArgs) {
+  return [{ title: data.documentTitle }];
+}
+
+export async function action({ context, request }: Route.ActionArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const lang = getLanguage(request);
+  const t = await getFixedT(request, handle.i18nNamespace);
+
+  const formData = await request.formData();
+  const action = formData.get('action');
+
+  switch (action) {
+    case 'back': {
+      throw i18nRedirect('routes/protected/person-case/privacy-statement.tsx', request);
+    }
+
+    case 'next': {
+      const schema = v.object({
+        scenario: v.pipe(
+          v.string(t('protected:request-details.required-scenario')),
+          v.picklist(VALID_SCENARIOS, t('protected:request-details.required-scenario')),
+        ),
+        type: v.pipe(
+          v.string(t('protected:request-details.required-request')),
+          v.picklist(VALID_REQUESTS, t('protected:request-details.required-request')),
+        ),
+      }) satisfies v.GenericSchema<RequestDetailsSessionData>;
+
+      const input = {
+        scenario: formData.get('scenario') as string,
+        type: formData.get('request-type') as string,
+      } satisfies Partial<RequestDetailsSessionData>;
+
+      const parseResult = v.safeParse(schema, input, { lang });
+
+      if (!parseResult.success) {
+        return data({ errors: v.flatten<typeof schema>(parseResult.issues).nested }, { status: 400 });
+      }
+
+      context.session.inPersonSINCase ??= {};
+      context.session.inPersonSINCase.requestDetails = parseResult.output;
+
+      throw i18nRedirect('routes/protected/person-case/first-name.tsx', request);
+    }
+
+    default: {
+      throw new AppError(`Unrecognized action: ${action}`);
+    }
+  }
+}
+
+export default function CreateRequest({ loaderData, actionData, params }: Route.ComponentProps) {
+  const { t } = useTranslation(handle.i18nNamespace);
+  const fetcher = useFetcher<Info['actionData']>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const errors = fetcher.data?.errors;
+  const scenarioOptions = VALID_SCENARIOS.map((value) => ({
+    value: value,
+    children: t(`protected:request-details.scenarios.${value}` as 'protected:request-details.scenarios.for-self'),
+    defaultChecked: value === loaderData.defaultFormValues.scenario,
+  }));
+  const requestOptions = ['select-option', ...VALID_REQUESTS].map((value) => ({
+    value: value,
+    children: t(`protected:request-details.requests.${value}` as 'protected:request-details.requests.select-option'),
+  }));
+
+  return (
+    <div className="mb-8">
+      <PageTitle subTitle={t('protected:in-person.title')}>{t('protected:request-details.page-title')}</PageTitle>
+      <fetcher.Form method="post" noValidate>
+        <div className="space-y-10">
```

Comment:
Only wrap inputs with this div and change space between `6` class.

```suggestion
        <div className="space-y-6">
```